Etabs_Adapter/CRUD/Create/Bar.cs

```diff
@@ -199,7 +219,15 @@ private void SetSection(ConcreteSection section)
 
         private void SetSection(ExplicitSection section)
         {
-            m_model.PropFrame.SetGeneral(section.Name, section.Material.Name, section.CentreZ * 2, section.CentreY * 2, section.Area, section.Asy, section.Asz, section.J, section.Iy, section.Iz, section.Wply, section.Wplz, section.Wely, section.Wely, section.Rgy, section.Rgz);
+            m_model.PropFrame.SetGeneral(section.Name, section.Material.Name, section.Vz + section.Vpz, section.Vy + section.Vpy, section.Area, section.Asy, section.Asz, section.J, section.Iy, section.Iz, section.Wply, section.Wplz, section.Wely, section.Wely, section.Rgy, section.Rgz);
```

Comment:
Found some errors in here, not induced by you additions, but that need to be fixed.

Section modulus `S` should be relating to `Wel`, right now Wpl is used.
Plastic modulus `Z` should relate to `Wpl`, right now `Wel` is used. `Wely` is also used twice.

Need to double check directions here as well, to make sure we have the right conversions from Iy/Iz to I22/33 as Etabs deals with this a bit differently than most other packages. From example section:

![image](https://user-images.githubusercontent.com/22005920/73173734-592dbb80-4106-11ea-9794-7399e1d7d39c.png)

I33 > I22 hence I33 = Iy in BHoM. and similarly S33 = Wely, Z33 = Wply and R33 = Rgy.

Same logic As2 = Asz and As3 = Asy